docs/intro.asciidoc

```diff
@@ -69,3 +69,29 @@ https://unpkg.com/@elastic/apm-rum/dist/bundles/elastic-apm-rum.umd.min.js[UNPKG
 ----
 
 NOTE: Currently our minified JavaScript bundle is about 16KB (gzipped).
+
+[float]
+[[Configuring CORS]]
+=== Configuring CORS
+
+If APM Server is deployed in an origin different than the page's origin,
+you will need to configure Cross-Origin Resource Sharing (CORS).
+
+TIP: To learn more about CORS, and why this process is necessary, see the MDN page on
+https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS[Cross-Origin Resource Sharing].
+
+To configure CORS,
+//configuration goes here
+
+When using this setup,
+the RUM agent will send an `HTTP OPTIONS` request prior to making the original `POST` request.
```

Comment:
Technically the RUM agent only makes the original POST request, however, since the browser checks if this request is a cross-origin request, if it is (e.g. in this case) the browser automatically makes an OPTIONS request to the server to make sure the original POST request is allowed by the server and if this check passes then the browser will send the original POST request. From the RUM agent's perspective we only make one http request with method set to POST.

The function of the OPTIONS request (also called preflight request) is to ensure that the server knows about the request's origin and that the server is prepared to accept the request with the specified method and headers.